lib/mutant/ast/regexp/transformer/direct.rb

```diff
@@ -0,0 +1,97 @@
+module Mutant
+  module AST
+    module Regexp
+      class Transformer
+        class Direct < self
+          register :regexp_one_or_more_escape
+          register :regexp_group_open_escape
+          register :regexp_group_close_escape
+          register :regexp_interval_open_escape
+          register :regexp_interval_close_escape
+          register :regexp_newline_escape
+          register :regexp_zero_or_more_escape
+          register :regexp_carriage_escape
+          register :regexp_dot_escape
+          register :regexp_set_open_escape
+          register :regexp_set_close_escape
+          register :regexp_eol_escape
+          register :regexp_bos_anchor
+          register :regexp_match_start_anchor
+          register :regexp_word_boundary_anchor
+          register :regexp_eos_ob_eol_anchor
+          register :regexp_eos_anchor
+          register :regexp_bol_anchor
+          register :regexp_eol_anchor
+          register :regexp_nonword_boundary_anchor
+          register :regexp_alpha_property
+          register :regexp_script_arabic_property
+          register :regexp_script_hangul_property
+          register :regexp_script_han_property
+          register :regexp_script_hiragana_property
+          register :regexp_script_katakana_property
+          register :regexp_letter_any_property
+          register :regexp_digit_type
+          register :regexp_space_type
+          register :regexp_word_type
+          register :regexp_nondigit_type
+          register :regexp_nonspace_type
+          register :regexp_nonword_type
+          register :regexp_dot_meta
```

Comment:
@backus What about a `%[regexp_a regexp_b ...].each(&method(:register))` to avoid the duplication? Should also fix the flog score.